nncf/common/graph/operator_metatypes.py

```diff
@@ -152,6 +154,23 @@ def get_input_metatypes() -> List[Type[OperatorMetatype]]:
     return input_metatypes
 
 
+def get_noop_metatypes() -> List[Type[OperatorMetatype]]:
+    """
+        Returns a list of the input operator metatypes.
+
+        :return: List of the input operator metatypes .
+    """
```

Comment:
```suggestion
    """
    Returns a list of the input operator metatypes.

    :return: List of the input operator metatypes .
    """
```

Reply:
We really need to use `black` in the future to fix this for us and save reviewer/author time